aws/ec2/install

```diff
@@ -12,24 +12,32 @@ curl --silent --location https://rpm.nodesource.com/setup_8.x | sudo bash -
 
 # Install dependencies with Yum
 sudo yum -y update
-sudo yum -y install git jq nodejs
-
-# Install RVM
-gpg --batch --list-keys # Initialize ~/.gnupg/
-gpg --batch --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
-curl -sSL https://get.rvm.io | bash -s stable
-source .rvm/scripts/rvm
-rvm install "$RUBY"
-rvm use     "$RUBY" --default
-# In order to use RVM outside of this script, the user will either need to
-# logout and login again, or run `source ~/.rvm/scripts/rvm` in the shell
+sudo yum -y groupinstall "Development Tools"
+sudo yum -y install git jq nodejs openssl-devel readline-devel
+
+# Install rbenv
+export PATH="$HOME/.rbenv/bin:$PATH"
+echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bash_profile
+curl -fsSL https://github.com/rbenv/rbenv-installer/raw/master/bin/rbenv-installer | bash
+rbenv install 2.5.1
+ruby global 2.5.1
+~/.rbenv/bin/rbenv init
```

Comment:
I know this is not how you're doing things, but one suggestion would be to create a Docker image you can spin up to do this work.

```Dockerfile
FROM ruby:2.5.1

RUN apt-get install -y git-core ...

RUN curl -L rbspy...

etc ...
```

It's a debian-based image, but it's transferrable so you and I can run these on development machines. If this is of interest to you, I can try my hand at converting this file into a Dockerfile at some point? Up to you 😄